pix-editor/app/components/sidebar/main.js

```diff
@@ -4,7 +4,7 @@ import ENV from 'pixeditor/config/environment';
 
 export default class SidebarMain extends Component {
   version = ENV.APP.version;
-
+  pix1dFramework = 'Pix 1D';
```

Comment:
proposition de renommage 
```suggestion
  pix1dFrameworkName = 'Pix 1D';
```